### PR TITLE
Bosch BMI160 driver bugfixes

### PR DESCRIPTION
**Pull request by NaterGator**
This addresses a couple of bugs with the BMI160 driver:
* 2000dps scale gyro register set incorrectly
* SPI bus clock running inappropriately slowly
* Overzealous accel/gyro publishing rate

### CODE REVIEW

File: src/drivers/bmi160/bmi160.hpp [hunk 1]
```diff
@@ -227,8 +227,10 @@
 #define BMI160_GYRO_DEFAULT_RANGE_DPS		2000
 #define BMI160_ACCEL_DEFAULT_RATE           800
 #define BMI160_ACCEL_MAX_RATE               1600
```
Comment (julianoes) [inline src/drivers/bmi160/bmi160.hpp]: This define seems now unused? Can it be removed?

Comment (NaterGator) [inline src/drivers/bmi160/bmi160.hpp]: That's true; that value does correctly specify the maximum sample rate of the accel per the datasheet, but the ioctl for max rate uses the faster gyro rate to set the hrt callback. I have no strong opinion on removing it; happy to defer.

Comment (julianoes) [inline src/drivers/bmi160/bmi160.hpp]: I see, if it's correct with the datasheet, that seems ok to leave it in there.

**Comment by julianoes** [comment]
@NaterGator I'm assuming this is tested? On which device could we test it?

**Comment by NaterGator** [comment]
Yes it has been tested; we're using it on a number of development boards. I'll try to grab a log with both the BMI160 and MPU6000 side-by-side.

**Pull request merged**